### DISCUSSION TITLE
Mesh for rice

### DISCUSSION THREAD

**Discussion by Schattenhammer** [vectorwing/FarmersDelight#917] [Ideas]
Hey guys :)

I love rice in the game. I got a big waterfield in my greenhouse with some bees.
Sadly the Bees drown pretty often while pollinating the rice. 

Could you make some mesh / net to lay it on the water (like Lilypads) where rice could 
grow through, but you can walk over it and bees can't drown in it?

**Comment by vectorwing** [comment]
Oof, that's a difficult thing to design and implement, sorry. 😅
I heard that 1.20.4 or 1.21 did some fixes to bee AI, which should hopefully improve their navigation around water a bit. There might also be a few mods which do specifically that, if I can recall.
